File: DevExpress-Demo/CS/ChartsDemo.Wpf/DataModels/LondonTemperatureData.cs
using System;
using System.Collections.Generic;
using System.Xml.Serialization;

namespace ChartsDemo {

    [XmlRoot("TemperatureData")]
    public class TemperatureData : List<TemperaturesInfo> {
        public static TemperatureData Load(){
            XmlSerializer serializer = XmlSerializer.FromTypes(new[] { typeof(TemperatureData) })[0];
            return (TemperatureData)serializer.Deserialize(DataLoader.LoadFromResources("/Data/LondonTemperature.xml"));
        }
    }


    public class TemperaturesInfo {
        public string Description { get; set; }
        public List<TemperatureInfo> Items { get; set; }
    }


    public class TemperatureInfo {
        public DateTime Date { get; set; }
        public int Temperature { get; set; }
    }

}

File: DevExpress-Demo/VB/ChartsDemo.Wpf/DataModels/LondonTemperatureData.vb
Imports System.Collections.Generic
Imports System.Xml.Serialization

Namespace ChartsDemo

    <XmlRoot("TemperatureData")>
    Public Class TemperatureData
        Inherits List(Of TemperaturesInfo)

        Public Shared Function Load() As TemperatureData
            Dim serializer As XmlSerializer = XmlSerializer.FromTypes({GetType(TemperatureData)})(0)
            Return CType(serializer.Deserialize(LoadFromResources("/Data/LondonTemperature.xml")), TemperatureData)
        End Function
    End Class

    Public Class TemperaturesInfo

        Public Property Description As String

        Public Property Items As List(Of TemperatureInfo)
    End Class

    Public Class TemperatureInfo

        Public Property [Date] As Date

        Public Property Temperature As Integer
    End Class
End Namespace
